README.md

```diff
@@ -31,13 +31,13 @@ Now follow the upcoming instructions.
 
 ```sh
 # Clone the project:
-$ git clone git@github.com:serlo-org/athene2.git
+$ git clone https://github.com/serlo-org/athene2.git
```

Comment:
IMHO: SSH is preferred as soon as you need to push at some point since you have to reenter your credentials everytime w/ HTTPS. Did you run into issues w/ SSH?